### PR TITLE
Add missing endpoints and restructure logic

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Refactored the NetworkInfo component and added missing endpoints

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/build/_partials/_oracles_contract_data.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { AddToMetaMaskButton } from '@theme/AddToMetaMaskButton';
+import { Networks } from '@theme/constant';
```
Comment (jlvandenhout) [inline docs/build/_partials/_oracles_contract_data.md:2]: Why is this here? I don't see it used. Also shouldn't this be an MDX file, because we are importing?

Comment (Dr-Electron) [reply docs/build/_partials/_oracles_contract_data.md:2]: Yes and yes

Comment (Dr-Electron) [reply docs/build/_partials/_oracles_contract_data.md]: Fixed that in another [PR](https://github.com/iotaledger/iota-wiki/pull/1604)

**Review by lucas-tortora — APPROVED** [review]
